FRAuth/FRAuth/Authentication/AuthService.swift

```diff
@@ -18,7 +18,7 @@ import FRCore
     * An error, if occurred during the authentication flow
  
  ## Notes ##
-     * Any Callback type returned from OpenAM must be supported within CallbackFactory.shared.supportedCallbacks.
+     * Any Callback type returned from AM must be supported within CallbackFactory.shared.supportedCallbacks.
```

Comment:
Update year on the header for modified files?

Reply:
Thanks for pointing that out @rodrigoareis . Updated all files in this PR.